### ISSUE TITLE
URLs no longer operational?

### ISSUE THREAD

**Issue by ssaxe-usgs** [USGS-R/dataRetrieval#505]
Are URLs constructed in `constructNWISURL()` no longer valid for accessing NWIS streamflow dvs?

The URL created with:
 `constructNWISURL(siteNumbers = "01387400", parameterCd = "00060", startDate = "1952-02-01", endDate = "2020-02-01")`
results in a "no applicable method for 'http_error'..." error from with the `readNWISdv()`  function.  The output URL from this function is:

> https://waterservices.usgs.gov/nwis/dv/?site=01387400&format=waterml,1.1&ParameterCd=00060&StatCd=00003&startDT=1952-02-01&endDT=2017-01-01

which leads to an XML page.

Through the NWIS site, the resulting URL for daily values for the same arguments is: 

> https://nwis.waterdata.usgs.gov/usa/nwis/dv/?cb_00060=on&format=rdb&site_no=01387400&period=&begin_date=1952-02-01&end_date=2020-02-21

I am using:
`packageVersion("dataRetrieval")`
`‘2.7.5.9000’`

**Comment by ldecicco-USGS** [comment]
`readNWISdv` does use the XML services (WaterML 1). That was a design decision years ago because it's theoretically a safer retrieval (there's no chance to get an incomplete record, and a few other reasons). 
I just ran:
```r
library(dataRetrieval)
x <- readNWISdv(siteNumbers = "01387400", parameterCd = "00060", 
                 startDate = "1952-02-01", endDate = "2020-02-01")
```
and got 14829 rows back, so it works for me.

My first recommendation would be to make sure you've got all your packages up-to-date:
![update](https://user-images.githubusercontent.com/1105215/75043898-59f80a00-5486-11ea-8216-2c95d8b4bf26.png)

The error you are describing sounds like something that was fixed...maybe...in the `httr` package awhile back (? I don't remember exactly, might ...but I think I remember seeing it). Also, I'd recommend checking if you are on R version 3.6. CRAN maintains packages only for the current major version (3.6.X) and one below (3.5.x). So, if you are on 3.4, there might be packages that have updates that you aren't automatically getting.
